src/components/portfolio/DetailsDonut.tsx

```diff
@@ -93,48 +121,58 @@ const DetailsDonut: React.FC<DetailsDonutProps> = ({
           chart: {
             height: size,
           },
-          legend: {
-            show: true,
-            position: 'right',
-          },
-        },
-      },
-      {
-        breakpoint: 1550,
-        options: {
-          chart: {
-            height: size / 1.5,
-          },
-          legend: {
-            show: true,
-            position: 'right',
-          },
         },
       },
       {
-        breakpoint: 1100,
+        breakpoint: 1200,
         options: {
           chart: {
-            height: size / 1.5,
-          },
-          legend: {
-            show: showLegendOnScale,
-            position: 'bottom',
+            height: size / 1.2,
           },
         },
       },
     ],
   };
 
+  // not including 'other ...'
+  const legendLength = 5;
+
+  const legendItems: JSX.Element[] = [];
+
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < labels.length; i++) {
```

Comment:
Even though I don't understand why linter would cry here you could use 'i+=1' instead to get rid of the eslint-disable.